spec/system/bus_stops/editing_bus_stops_spec.rb

```diff
@@ -0,0 +1,103 @@
+
+require 'spec_helper'
+
+describe 'editing a bus stop as a user' do
+  let!(:user) { create :user }
+  let!(:edit_stop) { create :bus_stop }
+  before :each do
+    when_current_user_is user
+    visit edit_bus_stop_path(edit_stop.hastus_id)
+  end
+  context 'with no changes' do
+    before :each do
+      within 'table.edit-form' do
+        click_button 'Save stop'
+      end
+    end
+    it 'updates the stop' do
+      expect(page).to have_selector 'p.notice',
+                                    text: 'Bus stop was updated.'
+    end
+    it 'redirects to the bus stops page' do
+      expect(page.current_url).to end_with bus_stops_path
+    end
+  end
+  context 'with data changes' do
+    before :each do
+      within 'table.edit-form' do
+        select 'UMTS', from: 'Garage responsible'
+        click_button 'Save stop'
+      end
+    end
+    it 'updates the stop' do
+      expect(page).to have_selector 'p.notice',
+                                    text: 'Bus stop was updated.'
+    end
+    it 'redirects to the bus stops page' do
+      expect(page.current_url).to end_with bus_stops_path
+    end
+  end
+  #this context block is failing. TODO: figure out why
```

Comment:
What kind of 'failing' is it?

Reply:
```
 Failure/Error: check 'Completed'

     Selenium::WebDriver::Error::UnknownError:
       unknown error: Element <input class="triple-zoom" type="checkbox" value="1" name="bus_stop[completed]" id="bus_stop_completed"> is not clickable at point (270, 131). Other element would receive the click: <label for="bus_stop_has_power">...</label>
         (Session info: chrome=67.0.3396.99)
         (Driver info: chromedriver=2.35.528157 (4429ca2590d6988c0745c24c8858745aaaec01ef),platform=Mac OS X 10.12.5 x86_64)
```

Reply:
Weird. It's trying to click on the checkbox, by clicking something else entirely?

Is this a byproduct of our CSS?

Reply:
I think it is because of the CSS. I can't say for sure though.

Reply:
Have you tried removing the `triple-zoom` class? I'm curious as to why that would cause the issue.

Reply:
Yeah, it fixes the problem when I remove the `triple-zoom` class

Reply:
It's the `zoom` property. I'll use `transform: scale` from now on.